source/_components/climate.toon.markdown

```diff
@@ -13,6 +13,15 @@ logo: toon.png
 ha_iot_class: "Cloud Polling"
 ---
 
-The `toon` climate platform allows you to interact with your Toon device.
+The `toon` climate platform allows you to interact with your Toon thermostat. For compatibility reasons the states are different from the normal "Toon programma's", they are mapped as follows:
```

Comment:
I like the addition but maybe we should keep the documentation as English as possible.
I like to suggest: `For compatibility reasons, the states in Home Assistant are different from the states displayed on your Toon device and are mapped as follows:`